packages/converters/src/to-boolean/index.test.ts

```diff
@@ -10,34 +10,21 @@
  * governing permissions and limitations under the License.
  */
 
-import { expect, it, describe } from 'vitest';
-import { toBoolean } from './';
+import { describe, expect, it } from 'vitest';
 
-const truthy = [1, '1', 'on', 'true', 'yes', true, 'ON', 'YES', 'TRUE'];
-const falsey = [
-  0,
-  '0',
-  'off',
-  'false',
-  'no',
-  false,
-  [],
-  {},
-  'OFF',
-  'NO',
-  'FALSE',
-];
+import { createFixture, falsey, truthy } from '../../../__fixtures__/boolean';
 
-describe('toBoolean', () => {
-  for (const item of truthy) {
-    it(`should return true for ${item}`, () => {
-      expect(toBoolean(item)).toBeTruthy();
-    });
-  }
+import { toBoolean } from './';
 
-  for (const item of falsey) {
-    it(`should return false for ${item}`, () => {
-      expect(toBoolean(item)).not.toBeTruthy();
+describe('toBoolean', () => {
+  describe.each([
+    // positive cases
+    ...createFixture(true, [truthy, toBoolean]),
+    // negative cases
+    ...createFixture(false, [falsey, toBoolean]),
+  ])('%s', (_name, expected, fn, values) => {
```

Comment:
Yea not just export this completed from the fixture? Test should minimize as much logic as possible.

Reply:
It's custom per test; `createFixture` is used in more than one place.

Reply:
Should I move it down and reuse across modules at the lower level?

Reply:
Personally, I subscribe heavily to [the kind of "go-to" best practices doc](https://github.com/goldbergyoni/javascript-testing-best-practices?tab=readme-ov-file#section-0%EF%B8%8F%E2%83%A3-the-golden-rule) so I don't really consider stuff like that needed. Which dovetails into the other discussion of making it dead simple, colocated and easier to maintain.

Reply:
Duplicating code is a bad practice too, and worse than not having everything colocated. #IMHO

Reply:
This no longer exists after the refactor of `toBoolean`.